src/controller/index.ts

```diff
@@ -265,15 +284,16 @@ export default class SimulariumController {
         }
     }
 
-    public pause(): void {
+    public pauseStreaming(): void {
         if (this.simulator) {
+            this.handleStreamingChange(false);
+            // todo add frame argument once octopus supports this
             this.simulator.pause();
-            this.isPaused = true;
         }
```

Comment:
what's the use case for this being a public method? I'm trying to follow the logic but it feels circular. is the point to allow the app that installs this to turn on and off prefetching? if so, why wouldn't that just be controlled by the `disabledCache` prop? Or if it's controlled by a function, why does it also have to call this callback? Presumably the installing app called the function because its state changed, so it already knows that streaming is off. It seems like this is state is being stored in a lot of places